Frontend/CO.CDP.OrganisationApp/Pages/Registration/JoinOrganisation.cshtml

```diff
@@ -0,0 +1,102 @@
+                            <p class="govuk-body">John</p>
+                        </dd>
+                    </div>
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key">
+                            Lastname
+                        </dt>
+                        <dd class="govuk-summary-list__value">
+                            <p class="govuk-body">Johnson</p>
+                        </dd>
+                    </div>
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key">
+                            Email address
+                        </dt>
+                        <dd class="govuk-summary-list__value">
+                            <p class="govuk-body">john@johnson.com</p>
+                        </dd>
+                    </div>
+                </dl>
+            </fieldset>
+        </div>
+    </div>
+    <br/>
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <form method="post">
+                <fieldset class="govuk-fieldset" data-required="" data-error="Select yes if you want to join this organisation" aria-required="true">
+                    <div class="govuk-form-group @(joinHasError ? "govuk-form-group--error" : "")">
+                        <h2 class="govuk-heading-m">
+                            <span class="govuk-label--m">Would you like to join @organisationDetails.Name?</span>
+                        </h2>
+                        @if (joinHasError)
+                        {
+                            <p id="approval-error" class="govuk-error-message">
+                                <span class="govuk-visually-hidden">Error:</span>
+                                @Html.ValidationMessageFor(m => m.Join)
+                            </p>
+                        }
+                        <div class="govuk-radios govuk-radios--inline" data-module="govuk-radios">
+                            <div class="govuk-radios__item">
+                                <input class="govuk-radios__input" id="join-YES" name="Join" type="radio"
+                                       value="true">
+                                <label class="govuk-label govuk-radios__label" for="join-YES">Yes</label>
+                            </div>
+                            <div class="govuk-radios__item">
+                                <input class="govuk-radios__input" id="join-NO" name="Join" type="radio"
+                                       value="false">
+                                <label class="govuk-label govuk-radios__label" for="join-NO">No</label>
+                            </div>
+                        </div>
+                    </div>
+                </fieldset>
+                <div class="govuk-button-group govuk-!-margin-top-6">
+                    <govuk-button>
```

Comment:
We can do away with the button group and just put the margin top on the button.

Reply:
Done